sleap/gui/dialogs/frame_range.py

```
@@ -0,0 +1,42 @@
+
+        self.setWindowTitle(title)
+
+    def _update_max_frame_range(self, value):
+        min_frame_idx_field = self.form_widget.fields["min_frame_idx"]
+        max_frame_idx_field = self.form_widget.fields["max_frame_idx"]
+
+        max_frame_idx_field.setRange(value, max_frame_idx_field.maximum())
+
+    def _update_min_frame_range(self, value):
+        min_frame_idx_field = self.form_widget.fields["min_frame_idx"]
+        max_frame_idx_field = self.form_widget.fields["max_frame_idx"]
```

Ruff (0.8.2)

Local variable `max_frame_idx_field` is assigned to but never used

Remove assignment to unused variable `max_frame_idx_field`

(F841)